gradle/libs.versions.toml

```diff
@@ -1,39 +1,39 @@
 [versions]
 assertj = "3.23.1"
 # @pin
-aws2 = "2.17.224"
+aws2 = "2.17.267"
 bouncycastle = "1.70"
 docker = "3.2.13"
-grpc = "1.49.0"
-hoplite = "2.6.2"
-jackson = "2.13.3"
+grpc = "1.49.2"
+hoplite = "2.6.4"
+jackson = "2.13.4"
 jnrUnixsocket = "0.38.17"
-junit = "5.9.0"
-kotest = "5.4.2"
+junit = "5.9.1"
+kotest = "5.5.0"
 # @pin
 kotlin = "1.7.10"
-kotlinBinaryCompatibilityPlugin = "0.11.0"
-kotlinLogging = "2.1.23"
+kotlinBinaryCompatibilityPlugin = "0.11.1"
+kotlinLogging = "3.0.0"
```

Comment:
This looks like a big change. Are there release notes?

Reply:
https://github.com/MicroUtils/kotlin-logging/releases/tag/3.0.0

Reply:
Looked ok to me given our other dep upgrades recently.